.ruby-version

```diff
@@ -0,0 +1 @@
+2.3.0
```

Comment:
This is required for Netlify build to work, hope folks don't mind adding this little file. It will need to be kept in sync with website repo, but I don't suppose this will change too often. I've managed to work around having to add any of the other config files.